Multisig/Logic/Models/Address.swift

```diff
@@ -73,23 +99,13 @@ extension Address {
     }
 }
 
-extension Address: ExpressibleByStringLiteral {
-    public init(stringLiteral value: String) {
-        if let v = Address(value) {
-            self = v
-        } else {
-            assertionFailure("Invalid literal address: \(value)")
-            self = .zero
+extension EthereumAddress.Error: LocalizedError {
+    public var errorDescription: String? {
```

Comment:
"description"?

Reply:
the "errorDescription" is `LocalizedError`'s interface requirement